.eslintrc

```diff
@@ -11,6 +11,11 @@
     "no-var": "off",
     "indent": ["error", 4],
     "quotes": ["error", "double"],
-    "linebreak-style": "off"
+    "linebreak-style": "off",
```

Comment:
I dont think we should be removing any of these (Adding to warning is as good as removing people dont care about warnings). Max-len can be disabled at a place or two, but it is better to have it. It can be disabled you have a really really long string at some place.

Please can you share your motivation behind doing this. I am not able to find any reason as to why we are doing this ?